tests/regression/CMakeLists.txt

```diff
@@ -107,7 +107,8 @@ endif()
 
 add_test_pocl(NAME "regression/test_llvm_segfault_issue_889" COMMAND "test_llvm_segfault_issue_889")
 
-add_test_pocl(NAME "regression/test_issue_1711" COMMAND "test_issue_1711")
+# Also smoke test cq-tracing with this case.
```

Comment:
Just a nitpick (can be fixed in another next PR): maybe a few words on why this case in particular